src/styles/components/_catalog.scss

```diff
@@ -0,0 +1,7 @@
+.catalog {
+  max-width: 992px;
```

Comment:
Don't write magic numbers. It's always better to write the way you calculated this number with variables 
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Thanks for review! I'll fix it now.